packages/mask/src/extension/background-script/SocialNetworkService.ts

```diff
@@ -22,6 +27,28 @@ function requestSNSAdaptorPermission(ui: SocialNetworkUI.Definition) {
     return requestExtensionPermission({ origins: [...ui.declarativePermissions.origins] })
 }
 
+function requestSNSAdaptorsPermission(uis: SocialNetworkUI.Definition[]) {
+    const req = uis.filter((x) => !x.permission?.request())
```

Comment:
I made a check and found no SNS Adaptor is using `definition.permission.request`. I suggest to remove this field, and always use `declarativePermissions.origins`. That will allow us to skip the loading to the SNS Adaptor.